### ISSUE TITLE
fit_intercept option cannot be set to True

### ISSUE THREAD

**Issue by briandesilva** [dynamicslab/pysindy#37]
When a `SINDy` object is created using an optimizer with `fit_intercept` set to true, the individual estimators created by the `SINDy` object's `MultiOutputClassifer` have `fit_intercept=False`. Here's a minimal working example in two dimensions:
```
from numpy import ones
import pysindy as ps

x = ones((5, 2))

opt = ps.STLSQ(fit_intercept=True)
print('opt.fit_intercept:', opt.fit_intercept)

model = ps.SINDy(optimizer=opt)
model.fit(x)
for est in model.model.estimators_:
    print('inner_estimator.fit_intercept:', est.steps[1][1].fit_intercept)
```

I suspect this has something to do with the way `MultiOutputClassifier` is instantiating its sub-estimators.

**Comment by Ohjeah** [comment]
Parameters are not set properly in STLSQ: according to [sklearn docs](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html)

> All estimators should specify all the parameters that can be set at the class level in their __init__ as explicit keyword arguments (no *args or **kwargs).

However, `fit_intercept` is passed down as kwargs. Subsequently, calling `get_params` in `MultiOutputRegressor` won't pick it up and use the default value (False) when cloning the base estimator.

**Comment by briandesilva** [comment]
Fixed in #43 .
